### PR TITLE
Project view page

### PR DESCRIPTION
**Pull request by hys25**

### CODE REVIEW

File: src/pages/ProjectView.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+  const dispatch = useDispatch()
+  const { id: currentProjectId } = useParams()
+
+  useEffect(() => {
+    dispatch(getProject(currentProjectId))
+    return () => {
+      dispatch(reset())
+    }
+  }, [navigate, dispatch, currentProjectId])
+  const { project } = useSelector((state) => state.project)
```
Comment (ArturHorbenko) [inline src/pages/ProjectView.js]: Let's move it up to other inline hooks which returns value. It's good practice to group hooks which returns value, hooks which doesn't return anything ( like useEffect ), handlers

File: src/pages/ProjectView.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+      <ContentContainer>
+        <Title className="font-bold mb-[50px]">{project?.project_name}</Title>
+        <div className="flex flex-col mx-auto w-full text-white max-w-[700px]">
+          <img
+            alt="Project"
+            className="object-cover w-full h-auto mb-[30px] max-h-[450px]"
+            src={
+              project?.main_image_url
+                ? `${REACT_APP_BE_HOST}/${project?.main_image_url}`
+                : "https://blog.hubspot.com/hubfs/Team%20deciding%20on%20membership%20website%20builder.jpg"
```
Comment (ArturHorbenko) [inline src/pages/ProjectView.js]: Let's move string above component, and give it a name like `PLACEHOLDER`

File: src/pages/Projects.js [hunk 3]
```diff
@@ -1,73 +1,56 @@
...
     if (!isUser) {
       navigate("/auth/sign-in")
     }
-  })
+    dispatch(getProjects())
+    return () => {
+      dispatch(reset())
+    }
+  }, [navigate, dispatch])
+  const { projects } = useSelector((state) => state.project)
```
Comment (ArturHorbenko) [inline src/pages/Projects.js]: Same here, let's move it up to `useNavigate()`

File: src/pages/Projects.js [hunk 4]
```diff
@@ -1,73 +1,56 @@
...
+                key={_id}
+                style={{
+                  backgroundImage: `url(${
+                    main_image_url
+                      ? mainImageUrl
+                      : "https://blog.hubspot.com/hubfs/Team%20deciding%20on%20membership%20website%20builder.jpg"
+                  })`,
+                  backgroundPosition: "center center",
+                  backgroundRepeat: "no-repeat",
+                  backgroundSize: "cover",
```
Comment (ArturHorbenko) [inline src/pages/Projects.js]: Let's move other than `backgroundImage` styles to tailwind classes.

**Review by ArturHorbenko — COMMENTED** [review]
Hi, looks very good to me, I added small comments to improve readability.

**Comment by hys25** [comment]
@ArthurGorbenko thanks, pushed fix

**Comment by ArturHorbenko** [comment]
LGTM

**Pull request merged**